spec/unit/changes_table_populator/measure_created_or_updated_spec.rb

```diff
@@ -16,7 +16,7 @@
       before do
         measure = create :measure
 
-        db.run("UPDATE measures_oplog SET operation = 'C', operation_date = '#{Date.current}' " \
+        db.run("UPDATE measures_oplog SET operation = 'C', operation_date = '#{Time.zone.today}' " \
```

Comment:
These are broken in the original but this could do with being `Time.zone.today.to_formatted_s(:db)`

Currently this only works because the default date format in the app is the same as the `:db` format but if that gets changed all these specs will fail

Reply:
Will fix in a follow on PR